Innovative Flix/force-app/main/default/objects/Episode__c/validationRules/Description_minimum_length.validationRule-meta.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Description_minimum_length</fullName>
+    <active>true</active>
+    <errorConditionFormula>LEN( Description__c ) &lt;= 10 &amp;&amp; LEN( Description__c )&gt;0</errorConditionFormula>
```

Comment:
You don't have to check if it's longer than 0 character.

Reply:
Name changed